Disable The Customizable IndexPageURL

tharindu-b-hewage opened the pull request:
## Purpose
> Having a query parameter for a customized index page overrides the requested page URL from the browser cookie, which will not allow the user to be redirected to the previously accessed page. This PR fixes this issue.

darshanasbg commented on core/org.wso2.carbon.ui/src/main/java/org/wso2/carbon/ui/CarbonSecuredHttpContext.java:
```diff
@@ -178,16 +181,22 @@ public boolean handleSecurity(HttpServletRequest request, HttpServletResponse re
 
-        // Reading the requestedURL from the cookie to obtain the request made while not
-        // authanticated; and setting it as the indexPageURL
-        indexPageURL = CarbonUILoginUtil.getIndexPageUrlFromCookie(requestedURI, indexPageURL,
```
We need to preserve this capability..

Having custom index page is a one capability.
Ability to return, last accessed page is another capability. Seems like this code is handling this case as well.. Even we can remove the above functionality(having custom index page) based on the flag, we cannot remove this one.

tharindu-b-hewage replied:
Addressed in f42bc30b6acf6c38add848cc5253f24402ac6e90, by changing the approach.

Commit:
Remove custom index page redirection capability

This capability does override the index page URL from the cookie, which
will not allow user to be redirected to the previously accessed page.
